### ISSUE TITLE
Memory Tab is missing 

### ISSUE THREAD

**Issue by Timothybas** [shiruken/hackintosh#5]

**Comment by shiruken** [comment]
What memory tab are you referring to?

**Comment by Timothybas** [comment]
I mean this
![Screenshot 2020-03-27 at 9 05 55 AM](https://user-images.githubusercontent.com/39411908/77718982-3b97a800-700a-11ea-9c6e-7a168a6c719f.png)
Also, another issue is can't boot system with iGPU but using GPU0 I can boot normally no issue with that. os VDA decoder shows failed in Hackintosh tool

Here is the screenshot of that

![Screenshot 2020-03-27 at 9 08 57 AM](https://user-images.githubusercontent.com/39411908/77719251-dc866300-700a-11ea-8e91-4fb42de07870.png)

**Comment by Timothybas** [comment]
Do you have any idea why Intel generation showing this "???" ?

**Comment by shiruken** [comment]
Looks like the Memory tab in the About This Mac dialogue only shows up if the computer has user-upgradable memory, which the iMac19,1 model **does** support. You can fix this issue and restore the Memory tab by changing the `PlatformFeature` value on the SMBIOS tab of Clover Configurator from `0x22` to `0x22`. I'm not sure if this was intentional or just a bug in the model selector configuration. 

Source: https://www.insanelymac.com/forum/topic/341661-release-macos-catalina-10152/?page=2&tab=comments#comment-2700718

**Comment by shiruken** [comment]
My guide enables the iGPU in headless mode and can't be used for booting. You'll need to identify the correct value for the `AAPL,ig-platform-id` key that allows for full iGPU use _with your specific CPU_. This is also likely why VDA Decoding is not supported by your build, you have an [i5-8400](https://ark.intel.com/content/www/us/en/ark/products/126687/intel-core-i5-8400-processor-9m-cache-up-to-4-00-ghz.html), which has a different iGPU device ID (`0x3E92`) than the [i7-9700k](https://ark.intel.com/content/www/us/en/ark/products/186604/intel-core-i7-9700k-processor-12m-cache-up-to-4-90-ghz.html) in my system (`0x3E98`).

* [[Guide] Intel Framebuffer patching using WhateverGreen
](https://www.tonymacx86.com/threads/guide-intel-framebuffer-patching-using-whatevergreen.256490/)

**Comment by Timothybas** [comment]
Thanks, Memory Tab issue is fixed. Regarding iGPU headless mode. let me explain my issue in details

I using **nvidia geforce gt 710**  and its working properly without any issue. The problem only happens when I tried to enable iGPU headless mode.

When I enable Internal graphics in Bios and I plugged HDMI to my GT 710 system not booting up and showing me this error. (seems Boot not even started)

Note: I tried to connect my monitor to my onboard HDMI still the same issue persists and not able to boot.
![IMG_9666](https://user-images.githubusercontent.com/39411908/77817357-52b0c580-70f0-11ea-8e33-d96edf0f3d56.PNG)
![IMG_8398](https://user-images.githubusercontent.com/39411908/77817368-61977800-70f0-11ea-87d6-7bee1b2c2763.PNG)
